ruby_on_rails-postgres/README.md

```diff
@@ -0,0 +1,95 @@
+## Compose sample application
```

Comment:
Should be updated regarding the following comments on compose file and dockerfile.